arcgis_map_sdk/lib/src/arcgis_location_display.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:arcgis_map_sdk_platform_interface/arcgis_map_sdk_platform_interface.dart';
+
+class ArcgisManualLocationDisplay extends ArcgisLocationDisplay {
+  @override
+  String get type => "manual";
+
+  ArcgisManualLocationDisplay({super.mapId});
+
+  Future<void> updateLocation(UserPosition position) {
+    _assertAttached();
+    return ArcgisMapPlatform.instance
+        .updateLocationDisplaySourcePositionManually(
+      _mapId!,
+      position,
+    );
+  }
+}
```

Comment:
The use case for manual location displays is that an app might have an own location stream from a different source (such as geolocator) with specific settings.
Those locations are then process by my app and I would like to display the exact location that my app processes. For that I don't want arcgis to create a location client that fetches the position again, I want to show the location that I already retrieved.